adapters/brightroll/brightroll.go

```diff
@@ -55,6 +69,23 @@ func (a *BrightrollAdapter) MakeRequests(requestIn *openrtb.BidRequest, reqInfo
 		errors = append(errors, err)
 		return nil, errors
 	}
+
+	var account *Account
```

Comment:
Please consider extracting the account/publisher validation piece of code in its own separate method

Reply:
Will do the refactoring code as a separate PR. thanks.